data/events.toml

```diff
@@ -104,7 +104,25 @@
     video1Name = "Interview video"
     video1SocialLink = "https://www.youtube.com/watch?v=acDdmyJZW-U"
     video2Name = "Event video"
-    video2SocialLink = ""
+    video2SocialLink = "https://youtu.be/ElfToZ9EBpM"
```

Comment:
@purna135 
The thumbnail for Todd's interview is labeled "Sandra Meneses"

Reply:
could you share a screenshot?

Reply:
https://www.youtube.com/watch?v=P3GIrgYwsTI

Reply:
got it, thank you